packages/atoa_core/lib/src/models/store_details/store_details.dart

```diff
@@ -21,7 +21,7 @@ class StoreDetails with _$StoreDetails {
 }
 
 String? _locationNameFromJson(String? value) {
-  if (value == null || value == 'DEFAULT') {
```

Comment:
@omer-vancha how is this fixing the issue? `value == 'DEFAULT'` would be checked only when value is not null.